Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -1522,60 +1522,5 @@ public async Task ActionExecutionWithBranchpoint()
                 await StopAsync(seed);
             }
         }
-
-        [Theory]
```

Comment:
We don't need this test anymore?

Reply:
It's because `TrieStateStore` became to require blocks having `stateRootHash`. It makes them use genesis blocks having different hash. So this test isn't needed anymore. You can see [BlockPolicy<T>.ValidateNextBlock(...)][next-block]

[next-block]: https://github.com/planetarium/libplanet/pull/986/files/389ab877f9498e404837ea282be1ea4768fb0863#diff-2be0780c565436e7e7c378aba4a2cf5cR199-R217